### PR TITLE
use keyword for Camera tag added in circular buffer metadata

### PR DESCRIPTION
**Pull request by tlambert03**
The hardcoded string "Camera" seems to be a relatively important tag added to the circular buffer metadata during InsertImage (particularly in multi-camera mode), but it's hard-coded as "Camera" all over the codebase.  This adds a new device constant `g_Keyword_Metadata_CameraLabel` and uses it instead

### CODE REVIEW

File: DeviceAdapters/DahengGalaxy/ClassGalaxy.cpp [hunk 1]
```diff
@@ -144,19 +144,19 @@ int ClassGalaxy::Initialize()
 
 
         vectorDeviceInfo.clear();
-        //ö���豸
```
Comment (tlambert03) [inline DeviceAdapters/DahengGalaxy/ClassGalaxy.cpp:147]: hmmm... these unicode issues seem to have been unintentionally made simply by saving the file (in VSCode).   @marktsuchida, do you know how i can avoid this?

Comment (marktsuchida) [reply DeviceAdapters/DahengGalaxy/ClassGalaxy.cpp:147]: #469, in progress.

Comment (marktsuchida) [reply DeviceAdapters/DahengGalaxy/ClassGalaxy.cpp:147]: #469 has been merged.

Comment (tlambert03) [reply DeviceAdapters/DahengGalaxy/ClassGalaxy.cpp]: k updated here as well

**Comment by marktsuchida** [comment]
Thanks!

**Pull request merged**